src/librustdoc/html/render.rs

```diff
@@ -2287,11 +2287,29 @@ fn short_stability(item: &clean::Item, cx: &Context) -> Vec<String> {
 fn item_constant(w: &mut Buffer, cx: &Context, it: &clean::Item, c: &clean::Constant) {
     write!(w, "<pre class='rust const'>");
     render_attributes(w, it, false);
+
     write!(w, "{vis}const \
-               {name}: {typ}</pre>",
+               {name}: {typ}",
            vis = it.visibility.print_with_space(),
            name = it.name.as_ref().unwrap(),
-           typ = c.type_.print());
+           typ = c.type_.print(),
+    );
+
+    if c.value.is_some() || c.is_literal {
+        write!(w, " = {expr};", expr = c.expr);
+    } else {
+        write!(w, ";");
+    }
+
+    if let Some(value) = &c.value {
+        if value.to_lowercase() != c.expr.to_lowercase() &&
```

Comment:
Maybe assign these vars into temporary vars to avoid calling `to_lowercase` repeatably?

Reply:
👍